tools/buildsteps/windows/make-mingwlibs.sh

```diff
@@ -34,31 +64,30 @@ throwerror() {
   fi
 }
 
-setfilepath() {
-  FILEPATH=$1
-}
-
 checkfiles() {
   for i in $@; do
-    if [ ! -f "$FILEPATH/$i" ]; then
-      throwerror "$FILEPATH/$i"
+    if [ ! -f "$PREFIX/bin/$i" ]; then
+      throwerror "$PREFIX/bin/$i"
       exit 1
     fi
   done
 }
 
 buildProcess() {
-export PREFIX=/xbmc/project/BuildDependencies/mingwlibs/$ARCHITECTURE
+export PREFIX=/xbmc/project/BuildDependencies/mingwlibs/$TRIPLET
 if [ "$(pathChanged $PREFIX /xbmc/tools/buildsteps/windows /xbmc/tools/depends/target/*/*-VERSION)" == "0" ]; then
   return
 fi
 
-git clean -dffx $PREFIX
+if [ -d "$PREFIX" ]; then
+  rm -rdf $PREFIX/*/
```

Comment:
I meant why is `*/` needed? With this files in the `$PREFIX` directory won't get deleted.